test/simple/haskell-flake-test.cabal

```diff
@@ -14,6 +14,6 @@ executable haskell-flake-test
     main-is:          Main.hs
     build-depends:    
         base,
-        foo
+        foo >= 0.2
```

Comment:
Add a comment here saying why `>= 0.2` exists and, especially, how it relates to the jailbreak flag in the flake.

Reply:
https://github.com/srid/haskell-flake/pull/420/commits/1a4f4f6ea1ac6c87c238cbfd72cf8d2358b5a3ec